src/framework/lib/genesisframework/utils.rb

```diff
@@ -46,9 +47,19 @@ def self.log subsystem, message
         puts logline
         Syslog.open("genesis", Syslog::LOG_PID, Syslog::LOG_USER) unless Syslog.opened?
         Syslog.log(Syslog::LOG_INFO, logline)
-        if self.facter['asset_tag']
-          self.collins.log! self.facter['asset_tag'], message
+
+        # Load external logging modules and send log to them
+        if @@loggers.nil?
+          @@loggers = self.config_cache[:loggers].map {|logger|
+            begin
+              require "logging/#{logger.downcase}"
+              Logging.const_get(logger.to_sym)
+            rescue LoadError
+              puts "Could not load logger #{logger}"
```

Comment:
In my opinion, failing to load a logger isn't something that stops/should stop genesis from doing it's main task, running the selected target. The experience will be slightly degraded since the logs might not show up where you expect them to, but they'll still be around in the local logs.